GUI/utils/visualizer.py

```diff
@@ -7,7 +7,7 @@ class Visualizer(QWidget):
     def __init__(self, *args, **kwargs):
         super(Visualizer, self).__init__()
         hbox = QHBoxLayout(self)
-        pod = Pod()
-        hbox.addWidget(pod)
+        #pod = Pod()
+        #hbox.addWidget(pod)
```

Comment:
I believe this code you commented out to get the GUI to run on your laptop. You shouldn't commit this code because then the pod visualization will not appear for anyone else running the GUI.